src/js/index.js

```diff
@@ -0,0 +1,8 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import Component from '~/Component';
```

Comment:
Aren't `../../Component` and `./Component` themselves relative paths?

Do you mean that this plugin allows us to use paths relative to a some constant? Like no matter where in our directory we are, if we want to `import Component` then the path `from` which we import will always be the same? (e.g., `~/Component`)?